sdks/python/apache_beam/options/pipeline_options.py

```diff
@@ -680,6 +680,16 @@ def _add_argparse_args(cls, parser):
          'enabled with this flag. Please sync with the owners of the runner '
          'before enabling any experiments.'))
 
+    parser.add_argument(
+        '--number_of_worker_threads',
```

Comment:
It will be good to keep the name same as in JRH `getNumberOfWorkerHarnessThreads`

Reply:
Done. And also updated documentation about limited availability of the option (currently only when experiment 'use_unified_worker' is set).